.github/workflows/e2e_android.yml

```diff
@@ -109,27 +109,27 @@ jobs:
           avd-name: Pixel_4_XL_API_28
           script: yarn e2e:android-test --take-screenshots failing --updateSnapshot --headless
 
-      # - name: Inform Slack of Failure
-      #   id: post_failure
-      #   if: failure()
-      #   run: |
-      #     ts=$(curl -X POST -H 'Authorization: Bearer '"$SLACK_API_TOKEN"' ' \
-      #       -H 'Content-type: application/json' \
-      #       --data '{"channel":"'${{needs.start_slack_thread.outputs.channel_id}}'","thread_ts":"'${{needs.start_slack_thread.outputs.thread_ts}}'","text":"Failed! Please check workflow for results: https://github.com/department-of-veterans-affairs/va-mobile-app/actions"}' \
-      #       https://slack.com/api/chat.postMessage|
-      #       jq -r '.ts')
-      #     echo SLACK_THREAD_TS=${ts} >> $GITHUB_OUTPUT
-
-      # - name: Inform Slack of success
-      #   id: post_failure
-      #   if: ${{!failure()}}
-      #   run: |
-      #     ts=$(curl -X POST -H 'Authorization: Bearer '"$SLACK_API_TOKEN"' ' \
-      #       -H 'Content-type: application/json' \
-      #       --data '{"channel":"'${{needs.start_slack_thread.outputs.channel_id}}'","thread_ts":"'${{needs.start_slack_thread.outputs.thread_ts}}'","text":"Success! All E2E tests have passed"}' \
-      #       https://slack.com/api/chat.postMessage|
-      #       jq -r '.ts')
-      #     echo SLACK_THREAD_TS=${ts} >> $GITHUB_OUTPUT
+      - name: Inform Slack of Failure
```

Comment:
Same with this step, this should only run when the trigger is the cron

Reply:
added